jobs/k8s-e2e-external-storage.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- job:
+    name: k8s-e2e-external-storage
+    project-type: pipeline
+    concurrent: true
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph-csi
+      - build-discarder:
+          days-to-keep: 7
+          artifact-days-to-keep: 7
+    parameters:
+      - string:
+          name: k8s_version
+          default: '1.19.3'
```

Comment:
One small question, Do we wish to keep the default as 1.19.3 for now as it was having some issues with CI?

Reply:
uhm, right, that is meant to be 1.19.2